src/Main.elm

```diff
@@ -39,39 +40,60 @@ main =
 
 
 type alias Model =
-    Session Benzino.RoundDetails
+    { session : Session Benzino.RoundDetails
+    , initialRandomSeed : Time.Posix
+    }
 
 
 type Msg
     = PlayerWantsToBet Money
+    | FreshSeed Time.Posix
```

Comment:
Նույն տիպի տարբեր վարիանտները արժի իրար համահունչ պահած, ու համասեռ

> Parallel structure means that coordinate parts of a sentence, such as items in a series or list, have the same grammatical form. Items in a series must be all nouns, all verbs, or all participles, and so on.

Եթե հարցեր կան գնացեք կարդացեք
- https://writingcenter.gmu.edu/writing-resources/grammar-style/parallel-structure
- https://www.redwoodink.com/resources/how-to-create-clear-patterns-with-parallel-structure (բժիշկների մասով)

Reply:
> What is essential is that the local symmetries in these patterns play a decisive and quite unexpected role. Though hidden from view, they essentially control the way the pattern is seen and the way it works.

Գնացեք կարդացեք, եթե իհարկե հետաքրքիր ա

- http://nature-of-order.stefan-lesser.com/local-symmetries (ճարտարագիտական)

Reply:
Ինչքա՞ն համահունչ։ Ասենք բայ, թե՞ մեկը մի բան ուզում ա անի։

Reply:
Մտածում եմ բայ` անցյալ կատարյալ։ Մեսիջով կուրսի ա պահվում դեպքի մանրամասները

Reply:
Օրինակ կգրեք որ խնդրեմ

Reply:
հա․ ասենք խոսքի՝

```elm
type IncomingEvents 
  = SessionInitiated Time.Posix
  | DepositReceived Money
  | BetProposed Money
```